### DISCUSSION TITLE
[4.x] Selectable currency for money field

### DISCUSSION THREAD

**Discussion by lukas-jansen** [craftcms/cms#11047] [Ideas] [answered]
Add an option to the money field to change the currency of the field on entry/product level. I now did it by creating a custom drop-down field with all the currencies and used a condition rule on this field to show the right formatted money field but this is both for the user not the optimal solution as when you change the drop-down the value is gone off course and template wise this isn't also optimal.

**Comment by brandonkelly** [comment, marked as answer]
We originally planned on making it selectable, but it adds a ton of complexities to the field, which made it feel a bit out of scope for a built-in field type, and better left to a plugin. For example, it would make sorting by the field very difficult. We would need to start storing an additional normalized currency value for each field, based on currency exchange rates – which would need to be kept updated as the exchange rates change.
